src/components/molecules/Visualizer/Engine/Cesium/index.tsx

```diff
@@ -99,7 +104,39 @@ const Cesium: React.ForwardRefRenderFunction<EngineRef, EngineProps> = (
           {/* remove default double click event */}
           <ScreenSpaceEvent type={ScreenSpaceEventType.LEFT_DOUBLE_CLICK} />
         </ScreenSpaceEventHandler>
-        <Camera onChange={handleCameraMoveEnd} />
+        <ScreenSpaceCameraController
+          maximumZoomDistance={
+            property?.cameraLimiter?.cameraLimitterEnabled
+              ? property.cameraLimiter?.cameraLimitterTargetArea?.height || Number.POSITIVE_INFINITY
+              : Number.POSITIVE_INFINITY
```

Comment:
```suggestion
            property?.cameraLimiter?.cameraLimitterEnabled
              ? property.cameraLimiter?.cameraLimitterTargetArea?.height ?? Number.POSITIVE_INFINITY
              : Number.POSITIVE_INFINITY
```
Note that this impl implies camera limitter will be disabled when the camera height is zero